source/contracts/extensions/MarketFeeCalculator.sol

```diff
@@ -3,45 +3,70 @@ pragma solidity ^0.4.13;
 import 'reporting/IReputationToken.sol';
 import 'reporting/IUniverse.sol';
 import 'reporting/IReportingWindow.sol';
+import 'libraries/math/SafeMathUint256.sol';
+import 'reporting/Reporting.sol';
 
 
 contract MarketFeeCalculator {
+    using SafeMathUint256 for uint256;
+
     mapping (address => uint256) private shareSettlementPerEthFee;
     mapping (address => uint256) private validityBondInAttoeth;
+    mapping (address => uint256) private targetReporterGasCosts;
+
+    uint256 private constant DEFAULT_VALIDITY_BOND = 1 ether;
+    uint256 private constant FXP_TARGET_INDETERMINATE_MARKETS = 10 ** 16; // 1% of markets
```

Comment:
Recommend a divisor instead of fixed point.  e.g., `20` to be used as `num_markets/20`.  Naming is tricky, but the math ends up being easier and you don't need to worry about correctly doing `fxpMul`.

Reply:
I'm not sure how we would employ that value without assuming it is actually being used as `1 ether / DIVISOR`. The example `num_markets / 20` doesn't make sense to me based on the formula we decided to use.

Reply:
nevermind I get it, Will approach this next